tests/acceptance/features/apiDownloads/download.feature

```diff
@@ -60,7 +60,15 @@ Feature: Download file in project space
       | Brian |
 
 
-  Scenario: user with role viewer cannot get the old version of the file in the project space
+  Scenario: user with role viewer cannot download the old version of the file in the project space
     Given user "Alice" has uploaded a file inside space "download file" with content "new content" to "file.txt"
-    When user "Bob" tries to get version of the file "file.txt" with the index "1" of the space "download file" using the WebDAV API
+    And user "Alice" has uploaded a file inside space "download file" with content "newest content" to "file.txt"
+    When user "Bob" tries to download version of the file "file.txt" with the index "1" of the space "download file" using the WebDAV API
+    Then the HTTP status code should be "403"
+
+
+  Scenario: user with role viewer cannot get a list of versions of the file in the project space
+    Given user "Alice" has uploaded a file inside space "download file" with content "new content" to "file.txt"
+    And user "Alice" has uploaded a file inside space "download file" with content "newest content" to "file.txt"
+    When user "Bob" tries to get version of the file "file.txt" with the index 1 of the space "download file" using the WebDAV API
```

Comment:
```suggestion
    When user "Bob" tries to get the versions of the file "file.txt" from the space "download file" using the WebDAV API
```

Reply:
the step lists the versions of a file. so we should not have to mention the index